org-code-javabuilder/playground/src/main/java/org/code/playground/Playground.java

```diff
@@ -0,0 +1,5 @@
+package org.code.playground;
+
+public static final class Playground {
+  public static final Board board = new Board();
```

Comment:
Nice, I like this pattern - will be easier for implementation and mirrors Theater/Stage